geowebcache/pom.xml

```diff
@@ -229,6 +229,12 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.easymock</groupId>
```

Comment:
The classextension capability should be standard in EasyMock 3.2.  This artifact should just add a package named `classextension` to make old code work.

Reply:
I believe this has been addressed in later commits.